woocommerce/woocommerce-gutenberg-products-block
# Critical flows: Shopper → Checkout → Can view empty checkout message

Fixes #5770

This PR adds e2e tests to ensure that shoppers can see an empty cart message on the checkout page when there are no products in their cart.


### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e-dev -- tests/e2e/specs/shopper/checkout-empty.test.js` to run only this test case.
5. Make sure all tests pass.
